client/components/product-card/index.jsx

```diff
@@ -44,17 +47,19 @@ function ProductCard( {
 						<h3 className="product-card__title">{ title }</h3>
 					</div>
 				) }
-				<div className="product-card__header-secondary">
-					{ subtitle && <div className="product-card__subtitle">{ subtitle }</div> }
-					{ ! purchase && (
-						<ProductCardPriceGroup
-							billingTimeFrame={ billingTimeFrame }
-							currencyCode={ currencyCode }
-							discountedPrice={ discountedPrice }
-							fullPrice={ fullPrice }
-						/>
-					) }
-				</div>
+				{ showSecondaryHeader && (
+					<div className="product-card__header-secondary">
+						{ subtitle && <div className="product-card__subtitle">{ subtitle }</div> }
+						{ showPriceGroup && (
```

Comment:
There's no way we can display both elements since `showPriceGroup` depends on `subtitle` not being present. We give a priority to the `subtitle`.

It can go another way around, though: we can show neither the subtitle nor the price group.

I haven't constructed any additional condition like `showSubtitle` here, since there's no other factor than `subtitle` itself that would determine if it should be displayed or not.

Reply:
Thanks for elaborating! Although not super obvious, I confirm this looks good 👍

Reply:
@tyxla You're right, it's not super clear. I've just pushed a [minor update that should make the conditional rendering a bit more obvious](https://github.com/Automattic/wp-calypso/pull/38337/commits/8b317726255b9e84201d35e3f6291b90a510961d), I hope. Have another look.